agent/action/gold_verify.py

```
@@ -0,0 +1,207 @@
+    · **多个候选存活时取 det 框最宽的**，不是取数值最大的。金币串一定是这个窄 roi 里
+      最长的文本；而「取最大值」在低金币时会翻车 —— 真金币 900、噪声碎片读成 12345，
+      取最大就取到碎片了。
+    """
+    try:
+        ss = context.tasker.controller.post_screencap().wait().get()
+        if ss is None:
+            mfaalog.warning("[Gold] ⚠️ 截图失败，本次读数作废")
+            return None
+        reco = context.run_recognition(node, ss)
+        if not reco:
+            # 识别没跑起来（节点名错/被禁用/图像空）——与「跑了但没认出数字」不是一回事，
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)